ftl/node/main.py

```diff
@@ -46,9 +46,14 @@
     action='store',
     help='The path where the application data sits.')
 
+parser.add_argument("-v", "--verbose",
```

Comment:
nit: --verbosity as the long form?

Reply:
And variable name. You can use the "choices" parameter here to get the help text and validation for free.

Reply:
both done